practice/completer/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
```

Comment:
This could have a lang='en' included in the <html> tag.  Not sure who cares, but it seems standard practice.